lib/puppet/configurer.rb

```diff
@@ -65,8 +71,10 @@ def retrieve_catalog(query_options)
     end
 
     return nil unless result
-
-    convert_catalog(result, @duration)
+    converted = nil
+    convertTime = thinmark { converted = convert_catalog(result, @duration) }
```

Comment:
Nit picky comment: the preferred naming convention for local variables is to use `snake_case`.